### DISCUSSION TITLE
Asking about OT tokens in rinkeby

### DISCUSSION THREAD

**Discussion by PhyoLinMg** [smartcontractkit/full-blockchain-solidity-course-py#144] [Q&A] [answered]
At the video duration of 11:43:51, you mentioned OT tokens. How can I add this to metamask? Having fun with your course.
I am currently at lesson 12:Upgrades.

**Comment by PatrickAlphaC** [comment, marked as answer]
Awesome! Glad you're having fun. 

You can add it to metamask the same way you added LINK tokens. Get the address of the deployed OT token, and hit the little "add tokens" button in metamask and place the address there.

Let me know if that works.
